### DISCUSSION TITLE
Possible Dual core usage problem in version 1.22.0

### DISCUSSION THREAD

**Discussion by sailflipper** [micropython/micropython#13348] [Core Development] [answered]
I'm an old programer starting to tinker again with microprocessors.  I'm using a Pi Pico W with Thorny, I was trying to use dual core usage examples but the Pico kept crashing.  I tried the same program on a different Pico and it ran fine.  The difference between both Pico's was the loaded MicroPython version. The good version was 1.18.0 , the crashing version was the newly released version 1.22.0.  I loaded the new version over the older versions Pico and it demonstrated the same crashing behavior.  I then loaded the previous to new version 1.21.0 on both Pico's and they both run the dual core  demo ( from Simon Monk's book) fine.    From this, I would conclude there is a problem with 1.22.0.  I don't know who I should let know of this issue, so please forwarded this to whomever could fix or help.  R, Phil

**Comment by mendenm** [comment, marked as answer]
This was a known issue with 1.22.0.  see:
https://github.com/micropython/micropython/issues/13288
However, the current nightly build should include the fix.

**Comment by sailflipper** [comment]
Thank you!

Sail far and fast…   KB3WAF
